bootstrap.php

```diff
@@ -137,6 +140,33 @@ function nfd_create_event_queue_table() {
 	dbDelta( $sql );
 }
 
+/**
+ * Update affiliation code in option table
+ *
+ * @param object $container Container information
+ */
+function nfd_update_options_table( $container ) {
+	$brand_code = array(
+		'bluehost'        => '86241',
+		'hostgator'       => '57686',
+		'web'             => '86239',
+		'crazy-domains'   => '57687',
+		'hostgator-india' => '57686',
+		'bluehost-india'  => '86241',
+		'hostgator-latam' => '57686',
```

Comment:
HostGator LATAM requires an additional region check to differentiate from normal HostGator

Reply:
Is it necessary to verify additional regions because we have the same Jetpack affiliation code for all hosts, including Hostgator, Hostgator-India, and Hostgator-Latam?

Reply:
@manikantakailasa Ah, no. Let's remove the `hostgator-india`, `bluehost-india`, and `hostgator-latam` options then, as the other cases will cover these.

Reply:
removing hostgator-india, bluehost-india, and hostgator-latam will result in the brands mapping to default and not having their respective Jetpack affiliation codes.

$container->plugin()->id; container will return brand as hostgator-india, bluehost-india, and hostgator-latam

Reply:
@manikantakailasa The `container()->plugin()->id` will ONLY return the base brand depending on the plugin:

- bluehost
- hostgator
- crazy-domains
- web
- mojo

It is not configured to include any suffixes.